src/models/user.ts

```diff
@@ -13,6 +13,8 @@ const schema = Joi.object().keys({
   company: Joi.string().allow(null).default(null),
   rights: Joi.array().items(Joi.string()).required().example(['STATIC', 'USERS'])
     .description('See the getting started section about rights'),
+  isSecretActive: Joi.boolean().allow(null).required().example(true)
```

Comment:
Can you rename this to `twoFactorAuthenticationEnabled`?

Reply:
Updated